pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/Flake8Task.groovy

```diff
@@ -21,14 +21,46 @@ import org.gradle.process.ExecResult
 @CompileStatic
 public class Flake8Task extends AbstractPythonMainSourceDefaultTask {
 
-  public void preExecution() {
-    args(pythonDetails.virtualEnvironment.findExecutable("flake8").absolutePath,
-        "--config", "$component.setupCfg",
-        "$component.srcDir",
-        "$component.testDir")
-  }
-
-  @Override
-  void processResults(ExecResult execResult) {
-  }
+    public void preExecution() {
+        /*
+         Modified to only include folders that exist. if no folders exist, then
+         the task isn't actually run.
+          */
+        List<String> sArgs = [pythonDetails.virtualEnvironment.findExecutable("flake8").absolutePath,
+                     "--config", component.setupCfg]
+
+        if (!project.file(pythonDetails.virtualEnvironment.findExecutable("flake8").absolutePath).exists()) {
```

Comment:
It was very strange, i was getting integration tests that were failing with a "file not found".  started putting more and more checks into figure out what it coudn't find and magically started working.  i can start pulling some of that out and see if it breaks again